documentation/secure_storage.md

```diff
@@ -74,16 +74,17 @@ Below is an excerpt from the specification listing the most vital requirements:
    Typically, an implementation may rely on the REE for that purpose (protection
    level 100) or on hardware assets controlled by the TEE (protection level
    1000).
+   The current implementation does *not* provide any protection against
+   rollback, although most of the code needed to do so is already implemented.
```

Comment:
Shouldn't this state that current optee proposes `level 100`.
The last sentence is an invitation to more details...